commands/request.go

```diff
@@ -118,20 +119,34 @@ func obliviousDnsRequest(c *cli.Context) error {
 	dnsTypeString := c.String("dnstype")
 	targetName := c.String("target")
 	proxy := c.String("proxy")
+	configString := c.String("config")
 
 	var useproxy bool
 	if len(proxy) > 0 {
 		useproxy = true
 	}
 
-	odohConfigs, err := fetchTargetConfigs(targetName)
-	if err != nil {
-		return err
-	}
-	if len(odohConfigs.Configs) == 0 {
-		err := errors.New("target provided no valid odoh configs")
-		fmt.Println(err)
-		return err
+	var odohConfigs odoh.ObliviousDoHConfigs
+	var err error
+	if len(configString) == 0 {
```

Comment:
Minor suggestion: Can we use 
```suggestion
	if len(strings.TrimSpace(configString)) == 0 {
```
instead to handle the empty space string characters? `--config  <ss>  --domain <>` where `<ss> = spaces` results in `invalid byte: U+0020 ' '`